src/components/m-dropdown/index.scss

```diff
@@ -120,9 +146,13 @@
   }
 }
 
+.m-dropdown__toggle,
 .m-dropdown__link {
-  @include typo-text-longer();
+  @include typo-decls('typo-18');
```

Comment:
@jackblackCH 
We have other more semantice mixins for typography.
Please use them instead

Reply:
I understand.
May I ask you if you have studied the [typography guidelines](https://design.axa.com/web-guidelines/typography)?
If not please do so.
We don't want this meaningless and occasionally wrong numbers, but rather prefer semantic names.

Reply:
Okay, did you recognize that the `18` size changes for various breakpoints?
If not please read it mindfully.

And as said the reasoning is semantic code and readability:
> We don't want this meaningless and occasionally wrong numbers, but rather prefer semantic names.

All available mixins are here:
https://github.com/axa-ch/patterns-library/blob/develop/src/styles/typo/_typo-mixins.scss#L73-L253

Reply:
agree with @AndyOGo , please use one of the defined mixings

Reply:
I did not need the responsive font in this case, that was the reason I used `typo-18` over `typo-text-longer`